plugins/package-managers/pub/src/main/kotlin/Pub.kt

```diff
@@ -613,8 +626,7 @@ class Pub(
                         homepageUrl = homepageUrl,
                         // Pub does not create binary artifacts, therefore use any empty artifact.
                         binaryArtifact = RemoteArtifact.EMPTY,
-                        // Pub does not create source artifacts, therefore use any empty artifact.
-                        sourceArtifact = RemoteArtifact.EMPTY,
+                        sourceArtifact = sourceArtifact,
```

Comment:
What about the original comment here, was it wrong? Is it worth mentioning that in the commit message?

Reply:
I'm adding support for source artifacts, isn't it obvious that the sentence was wrong?

Reply:
The comment said "Pub does not create source artifacts" which to me sounded like Pub does not have the concept of source artifacts as all, so I was surprised that you can add support for them.

Reply:
TBH, I have no idea why that sentence was added in the first place, maybe it was wrong all the time, maybe it was correct back in 2019 when it was added.